.travis.yml

```diff
@@ -0,0 +1,26 @@
+sudo: required
+dist: trusty
```

Comment:
Have you had a look at the travis page which shows the running builds and its status? You'll see it is working :wink: . I took this script from the official ROS industrial repository. On the ROS wiki it is said that this is a very good entry point for prerelease testing. So it is tested quite well. Also you get a "build status" badge/banner which can be shown (I think I display this in the README.md).